Dockerfile

```diff
@@ -8,14 +8,17 @@ FROM ${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG}
 # ... then the modules installed during the transformation are not loaded automatically!
 # ... because the loader thinks that this directory does not exist (it did not exist at the start of the script)
 # Eg. mkdir -p /var/www/.local/lib/python3.8/site-packages
-RUN mkdir -p $(su www-data -s /bin/bash -c "python -c 'import site; print(site.USER_SITE)'")
+RUN . "$VIRTUAL_ENV/bin/activate" && mkdir -p $(su www-data -s /bin/bash -c "python -c 'import site; print(site.USER_SITE)'")
 
 # Make home directory writable
 RUN chown -R www-data:www-data /var/www
 
 WORKDIR /code/
-COPY ./src ./src
+
+COPY ./kbc_transformation ./kbc_transformation
 COPY ./tests ./tests
 COPY main.py README.md ./
 
-CMD ["python", "-X", "faulthandler", "-u", "/code/main.py"]
+RUN . "$VIRTUAL_ENV/bin/activate" && pip freeze
+
+CMD . "$VIRTUAL_ENV/bin/activate" && python -X faulthandler -u /code/main.py
```

Comment:
I'm always not sure about the shell vs exec form. Won't this break signal handling?

Reply:
the safest way would AFAIK be `CMD ["sh", "-c", ". "$VIRTUAL_ENV/bin/activate" && python -X faulthandler -u /code/main.py"]`, but according to docs, it should actually be the same as what you worte, so :crossed_fingers: :slightly_smiling_face: